src/chatwindow/chatdisabled.qml

```diff
@@ -0,0 +1,75 @@
+import QtQuick 2.8
+import QtQuick.Controls 2.0
+
+Item {
+    id: chatBody
+    visible: true
+    Rectangle {
+        anchors.fill: chatBody
+        color: {
+            if (globalTheme === "white")
+            {
+                return "#bde1ff"
+            }
+            if (globalTheme === "blue")
+            {
+                return "#2c8dde"
+            }
+            if (globalTheme === "dark")
+            {
+                return "#41607a"
+            }
+        }
+
+        Column {
+            anchors.centerIn: parent
+            //width: unsatisfiedUserImage.width
+            //spacing: 10
+            Control {
+                anchors.horizontalCenter: parent.horizontalCenter
+                //anchors.bottom: unsatisfiedUserImage.top
+                width: warningText.implicitWidth + 2
+
+                height: 30
+                Label {
+                    id: warningText
+                    text: qsTr("Log in to use chat")
+                    color: {
```

Comment:
You use this code twice, so it maybe better to make a special function for it

Reply:
Now these actions are preformed inside JS functions